src/index.html

```diff
@@ -8,6 +8,140 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars-star">
+
+        </div>
+
+        <div class="stars-star">
+
+        </div>
+
+        <div class="stars-star">
+
+        </div>
+
+        <div class="stars-star">
+
+        </div>
+
+        <div class="stars-star">
+
+        </div>
+      </div>
+
+      <div class="stars stars--1">
+        <div class="stars-star stars-star--active">
```

Comment:
```suggestion
      <div class="stars stars--1">
        <div class="stars__star">
```
Star elements shouldn't have any additional classes. You can get the active star according to the modifiers of `stars` block